internal/config/application.go

```diff
@@ -184,7 +184,7 @@ func (cfg Application) String() string {
 }
 
 // readConfig attempts to read the given config path from disk or discover an alternate store location
-// nolint:funlen
+
```

Comment:
(same two cents as in https://github.com/anchore/syft/pull/536#discussion_r723178891)